logo.py

```diff
@@ -1,2 +1,7 @@
 KamleshKumar Soni
 LOgo31 Adddee
+LOgo31 Adddee
+LOgo31 Adddee
+LOgo31 Adddee
+
+{'name': 'Soni'}
```

Comment:
Review start
`correct this`
```suggestion
{'name': 'Sandip Kumar soni'}
```